resources/settings.xml

```diff
@@ -66,12 +66,23 @@
           <level>0</level>
           <default>false</default>
           <control type="toggle"/>
+          <dependencies>
+            <dependency type="enable" operator="!is" setting="display_datetime">true</dependency>
+          </dependencies>
         </setting>
         <setting help="" id="display_current_week" label="30031" type="boolean">
           <level>0</level>
           <default>true</default>
           <control type="toggle"/>
         </setting>
+        <setting help="" id="display_datetime" label="30052" type="boolean">
+          <level>0</level>
+          <default>true</default>
+          <control type="toggle"/>
+          <dependencies>
+            <dependency type="enable" operator="!is" setting="hide_game_length">false</dependency>
+          </dependencies>
+        </setting>
```

Comment:
Hm, I don't really like cyclic dependencies, i.e. two items depending on each other. Is there a reason why you added the dependencies?

Reply:
The dependency is needed because if you do not hide the game length, the second line is used for game length and can no longer be used for datetime. 
So either you show game length in line 2 or datetime. If you wanted both, the layout would need adjusting. 
Alternativ was to create an either or dependency.

Reply:
Ah, I see. That makes sense indeed, I didn't have the layout in mind when checking the code. Thanks for the explanation